shinydashboard/lantern/fhir_endpoints_fields_json.md

```diff
@@ -23,7 +23,7 @@ The operation field is an array where an element is an instance of the informati
 * **tls_verison:** Transport Layer Security (TLS) version of the endpoint  
 * **mime_types:** MIME types supported by this endpoint  
 * **updated:** Timestamp of when this endpoint record was updated  
-* **supported_resources:** All of the FHIR resources this endpoint supports  
+* **operation_resource:** An array of pairs of a supported resource and one of it's supported operations.
```

Comment:
Can this change be reverted? Looks like the JSON export format hasn't changed.

Reply:
yep you're right, i think i got trigger happy with the find and replace, i've switched it back!